hudi-cli/src/main/java/org/apache/hudi/cli/commands/CompactionCommand.java

```diff
@@ -382,39 +375,36 @@ public String unscheduleCompactFile(
       @CliOption(key = {"desc"}, help = "Ordering", unspecifiedDefaultValue = "false") boolean descending,
       @CliOption(key = {"headeronly"}, help = "Header Only", unspecifiedDefaultValue = "false") boolean headerOnly)
       throws Exception {
+    HoodieTableMetaClient client = checkAndGetMetaClient();
```

Comment:
Did you get a chance to run any of these commands from command line to validate the changes? I know from the looks of it, this looks ok. But can you please run one compaction command from command line and ensure this code change looks good.

Reply:
> Did you get a chance to run any of these commands from command line to validate the changes? I know from the looks of it, this looks ok. But can you please run one compaction command from command line and ensure this code change looks good.

Yes, I had run and perform ok.

Reply:
ok